mesonbuild/backend/backends.py

```diff
@@ -636,6 +637,10 @@ def determine_ext_objs(self, extobj, proj_dir_to_build_root):
             unity_size = self.get_option_for_target(OptionKey('unity_size'), extobj.target)
 
             for comp, srcs in compsrcs.items():
+                if comp.language in ('d', 'fortran'):
+                    # See langs_cant_unity in ninjabackend.py
```

Comment:
Can we just move `langs_cant_unity` into the `backends` module and import it into `ninjabackend` instead of duplicating?

Reply:
Hey, I moved `ninjabackend.NinjaBackend.langs_cant_unity` into `backends.LANGS_CANT_UNITY` in the most recent revision, as you suggested.

Reply:
Thanks!